pkl-core/src/test/files/LanguageSnippetTests/input/basic/constModifier4.pkl

```diff
@@ -0,0 +1,11 @@
+const baz = 15
+
+foo {
+  const local bar = baz
```

Comment:
```suggestion
  const local bar = biz()
```

^^ otherwise; you don't cover the `const function` consumption side.

Reply:
That's covered on line 10 (see `corge = biz()`)

Reply:
Added another line to test calling a local method from a local property